diff --git a/third_party/libunwind/BUILD.gn b/third_party/libunwind/BUILD.gn
index ff1517d..875c813 100644
--- a/third_party/libunwind/BUILD.gn
+++ b/third_party/libunwind/BUILD.gn
@@ -21,7 +21,7 @@ config("libunwind_config") {
 
 # TODO(crbug.com/1458042): Move this build file to third_party/libc++/BUILD.gn once submodule migration is done
 source_set("libunwind") {
-  visibility = [ "//buildtools/third_party/libc++abi" ]
+  visibility = [ "//buildtools/third_party/libc++abi", "//build/config:common_deps" ]
   if (is_android) {
     visibility += [ "//services/tracing/public/cpp" ]
   }
